index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav id="navbar">
+        <ul>
+            <li><a href="#welcome-section">About</a></li>
+            <li><a href="#projects">Work</a></li>
+            <li><a href="#contact-section">Contact</a></li>
+        </ul>
+    </nav>
+
+    <section id="welcome-section">
+        <h1>Hey I am Mimic</h1>
+        <p>a web developer</p>
+    </section>
+
+    <section id="projects">
+        <div>
+        <div class="project-title"><h2>These are some of my projects</h2></div>
+    </div>
+    <div id="fleximg">
+        <div class="myprojects">
+        <a href="https://codepen.io/freeCodeCamp/full/zNqgVx">
+            <img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="">
+            <p class="project-name">Tribute Page</p>
+        </a>
+    </div>
+        <div class="myprojects"><a href="https://codepen.io/freeCodeCamp/full/qRZeGZ"><img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/random-quote-machine.png" alt=""><p class="project-name">Random Quote Machine</p></a></div>
+        <div class="myprojects"><a href="https://codepen.io/freeCodeCamp/full/wgGVVX"><img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/calc.png" alt=""><p class="project-name">Javascript Calculator</p></a></div>
+        <div class="myprojects"><a href="https://codepen.io/freeCodeCamp/full/mVEJag"><img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/map.jpg" alt=""><p class="project-name">Map Data Across the Globe</p></a></div>
+        <div class="myprojects"><a href="https://codepen.io/freeCodeCamp/full/wGqEga"><img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/wiki.png" alt=""><p class="project-name">Wikipedia Viewer</p></a></div>
+        <div class="myprojects"><a href="https://codepen.io/freeCodeCamp/full/KzXQgy"><img src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tic-tac-toe.png" alt=""><p class="project-name">Tic-Tac-Toe Game</p></a></div>
+    </div>
+    <div>
+     <button type="button" id="shows">Show All</button>
+    </div> 
+    </section>
+
+    <!-- <section id="footer">
+        <h1>Let's work together...</h1>
+        <p>How do you take your coffee?</p>
+        <u>
+        <li></li>
+        <li></li>
+        <li></li>
+        <li></li>
+        <li></li>
+        </u>
+    </section> -->
```

Comment:
Please fix the indentation of your code. To have clean code and for it to be easily readable, Have indentation of two spaces and not use the Tab key. Preferably change the number of spaces your Tab key gives in your text editor settings

Reply:
Very well-given review! :+1: :100:

Reply:
Thank you very much ❤️